docs/upgrade/index.rst

```diff
@@ -11,24 +11,30 @@ Upgrade ScyllaDB
    ScyllaDB Open Source to ScyllaDB Enterprise <upgrade-to-enterprise/index>
    ScyllaDB AMI <ami-upgrade>
 
+Overview
+---------
 
 ScyllaDB upgrade is a rolling procedure - it does not require a full cluster shutdown and is performed without any 
 downtime or disruption of service.
 
-To ensure a successful upgrade and avoid breaking anything, you should perform your upgrades consecutively - to each 
-successive X.Y version. For example, to upgrade from version 4.4 to 5.0, you should first upgrade ScyllaDB to version 
-4.5, next from 4.5 to 4.6, and finally, from 4.6 to 5.0, **without skipping any major version**.
+To ensure a successful upgrade, follow the :ref:`documented upgrade procedures <upgrade-upgrade-procedures>` tested by ScyllaDB. This means that:
 
-.. raw:: html
+* You should perform the upgrades consecutively - to each successive X.Y version, **without skipping any major or minor version**.
```

Comment:
What do you mean by not skipping any minor version?
Say one is upgrading from 2021.1.10 towards 2022.2.0, should one not skip any 2021.1.x version until the latest 2021.1.x (17?), then go through all 2022.1.x minor versions and only then upgrade to 2022.2.0?
Don't we just upgrade to the latest minor at each major version?

Reply:
No, the x in your example is the *patch* version.

> Given a version number MAJOR.MINOR.PATCH, increment the:
> MAJOR version when you make incompatible API changes
> MINOR version when you add functionality in a backwards compatible manner
> PATCH version when you make backwards compatible bug fixes
> 

https://semver.org/

Reply:
> No, the x in your example is the _patch_ version.
> 
> > Given a version number MAJOR.MINOR.PATCH, increment the:
> > MAJOR version when you make incompatible API changes
> > MINOR version when you add functionality in a backwards compatible manner
> > PATCH version when you make backwards compatible bug fixes
> 
> https://semver.org/

Maybe we should add a concrete example to clarify what we mean without having to resort to terms and definitions.

Reply:
@bhalevy I've added an example - it starts on line 28. Is that what you had in mind? I can add one more for Enterprise if you think that would be helpful.